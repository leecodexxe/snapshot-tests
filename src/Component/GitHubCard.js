import Button from 'react-bootstrap/Button';
import Card from 'react-bootstrap/Card';

function GitHubCard() {
  return (
    <Card style={{ width: '18rem' }}>
      <Card.Img variant="top" src="http://placekitten.com/g/200/300" />
      <Card.Body>
        <Card.Title>My Gitgub:leecodexxe</Card.Title>
        <Card.Text>
          Some quick example text to build on the card title and make up the
          bulk of the card's content.
        </Card.Text>
        <Button variant="primary">Go to My Github</Button>
      </Card.Body>
    </Card>
  );
}

export default GitHubCard;