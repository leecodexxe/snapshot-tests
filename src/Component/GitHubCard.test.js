import { render, screen } from '@testing-library/react'
import GitHubCard from './GitHubCard'
import renderer from 'react-test-renderer'

test('check for card for GitHub', () => {
    const tree = renderer.create(<GitHubCard/>).toJSON()
  expect(tree).toMatchSnapshot()
})