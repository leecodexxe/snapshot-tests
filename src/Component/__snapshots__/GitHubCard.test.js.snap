// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check for card for GitHub 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="http://placekitten.com/g/200/300"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      My Gitgub:leecodexxe
    </div>
    <p
      className="card-text"
    >
      Some quick example text to build on the card title and make up the bulk of the card's content.
    </p>
    <button
      className="btn btn-primary"
      disabled={false}
      type="button"
    >
      Go to My Github
    </button>
  </div>
</div>
`;
